### DISCUSSION TITLE
How to use tabs instead of spaces for indentation?

### DISCUSSION THREAD

**Discussion by ghost** [orbitalquark/textadept#371] [Q&A] [answered]
I want to use tabs instead of spaces and set tab width to 4 spaces. I discovered my text looks fine on TextAdept but if I opened on other text editors the indentation turned to be a mess. It seems TextAdept by default uses spaces for indentation, tab width is 2 spaces and tabs will be converted to spaces automatically.

Some text editors also has options to make spaces and tabs visible. If TextAdept also has these options please let me know too.

Thank you.

**Comment by orbitalquark** [comment, marked as answer]
Try setting `buffer.use_tabs = true` in your *~/.textadept/init.lua*: 
https://orbitalquark.github.io/textadept/manual.html#buffer-settings

You can toggle whitespace visibility with Buffer > Toggle View Whitespace.
